File: week11/week11-1.py
#week11-1 Hash Map, Hash Set
table = {} #Python 用大括號代表字典 也就是 Hash Table(Hash Map, Hash Set)
table[9977341] = 9
table[4433997] = 4

print(table[9977341]) #會印出9
print(table[4433997]) #會印出4
#print(table[0]) #糟! 沒有這key，爆炸
if 0 not in table:
  print("table[0]不存在")

File: week11/week11-1.cpp
///使用Hash Map, Hash Set，不能直接在CodeBlocks使用，因為是C++ 2021
///seting-compiler 加入C++11就行
#include <iostream>
#include <unordered_map>
using namespace std;
int main()
{
    unordered_map<int,int> table;
    table[9977341] = 9;
    table[4433997] = 4;
    cout<<table[9977341]<<endl;
    cout<<table[4433997]<<endl;
}
